CONTRIBUTING.md

```diff
@@ -82,13 +84,41 @@ Please be as descriptive as possible when creating an issue. Give us the informa
 - Make sure you have added the necessary tests for your changes.
 - Make sure you have documented your changes, if necessary.
 
-### Submitting changes as pull Requests
+### Submitting changes as Pull Requests
 
 - Push your changes to a topic branch in the fork of your repository. If you are working locally, do this with `git push -u my-gh-repo topic/my-fix`. `my-gh-repo` is typically `origin`; check with `git remote -v`.
 - Submit a pull request to the SuperCollider repository.
 - The core team looks at pull requests on a regular basis in a weekly meeting that we hold in a public meeting. The hangout is announced in the weekly status updates that are sent to the sc-dev list.
 - You may receive feedback and requests for changes. We expect changes to be made in a timely manner. We may close the pull request if it isn't showing any activity.
 
+### Updating your fork
+In order to keep your fork up-to-date, you need to point it to the main SuperCollider repository. This is done by adding the main repository as another `remote`, usually called `upstream`. **Please note:** naming the main repository `upstream` is just a convention, not a requirement. If you already have a differently named `remote` pointing to the main SuperCollider repository, you can use that name instead of `upstream`.
+- If you have not yet created the `upstream` `remote`, create it first:
+	- Check the list of remotes: `git remote -v`. The output should look like this:
+			origin	https://github.com/your-name/supercollider.git (fetch)
+			origin	https://github.com/your-name/supercollider.git (push)
+	- Add a new remote called `upstream`, pointing to the SuperCollider repository:
+	`git remote add upstream https://github.com/your-name/supercollider.git`
+	- Check the list of remotes again: `git remote -v`. Now the output should look like this:
+			origin	https://github.com/your-name/supercollider.git (fetch)
+			origin	https://github.com/your-name/supercollider.git (push)
+			upstream	https://github.com/supercollider/supercollider (fetch)
+			upstream	https://github.com/supercollider/supercollider (push)
```

Comment:
These lines aren't getting the right formatting: https://github.com/dyfer/supercollider/blob/b3aada7ec1288bd3e3fa62326b94b00859007238/CONTRIBUTING.md

I'm not 100% sure on how to get this but I think adding an empty line above and below your code blocks should be enough. See the working example in "making changes" above

Reply:
Thanks for catching that! Should work now...

Reply:
Hmm, still not good. I just tested it out a bit and it looks like the lines need to be indented by at least 14 spaces (no tabs). Annoying since the bulleted lists are in tabs...